scripts/check_change_log/main.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+import glob
+import os
+import subprocess as sp
+import time
+from pypi_tools.pypi import PyPIClient
+from pathlib import Path
+
+
+def find_report_name(result):
+    pattern = 'written to'
+    merged = 'merged_report'
```

Comment:
both var are pattern. I think it is better to be `single_api_pattern` and `multi_api_pattern`